features/doctrine/date-filter.feature

```diff
@@ -10,7 +10,7 @@ Feature: Date filter on collections
     Then the response status code should be 200
     And the response should be in JSON
     And the header "Content-Type" should be equal to "application/ld+json"
-    And the JSON should be equal to:
+    And the JSON should be valid according to this schema:
```

Comment:
I prefer using this method cause when I update configuration, I need to update every feature file about schema. There is no need on it as scenario just need to check configuration. Full schema is already tested on [`features/crud.feature`](https://github.com/teohhanhui/DunglasApiBundle/blob/filter_nested/features/crud.feature)